package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "sass-embedded",
-  "version": "1.72.0",
-  "protocol-version": "2.5.0",
-  "compiler-version": "1.72.0",
+  "version": "1.74.0",
```

Comment:
You don't need to update the version and compiler-version here manually, it's done as part of the release process.

Reply:
Ah. I got the error for the protocol version and just updated all of them. Fixed.